src/api/creatures.py

```diff
@@ -107,11 +106,11 @@ def get_creature_stats(user_id: int, creature_id: int):
         if not c_stats:
             raise HTTPException(
                 status_code=404,
-                detail=f"User {user_id} and/or creature {creature_id} does not exist.",
+                detail=f"User {user.id} and/or creature {creature_id} does not exist.",
```

Comment:
```suggestion
                detail=f"Creature {creature_id} does not exist.",
```

Reply:
Errors that include user and creature could be reworded to put an emphasis on the creature, but I don't want to completely delete the parts about the user